src/data/majors/cs.ts

```diff
@@ -25,35 +25,38 @@ const csRequirements: readonly CollegeOrMajorRequirement[] = [
   },
   {
     name: 'Computer Science Core',
-    description: 'CS 2800 or CS 2802, CS 3110, CS 3410 or CS 3420, CS 4410, and CS 4820',
+    description:
+      'CS 2800 or CS 2802, CS 3110, CS 3410 or CS 3420, CS 3700 or CS 3780, CS 4410, and CS 4820',
     source: 'https://www.cs.cornell.edu/undergrad/csmajor',
     checker: includesWithSubRequirements(
       ['CS 2800', 'CS 2802'],
       ['CS 3110'],
       ['CS 3410', 'CS 3420'],
+      ['CS 3700', 'CS 3780'],
       ['CS 4820'],
       ['CS 4410', 'CS 4414']
     ),
     fulfilledBy: 'courses',
-    perSlotMinCount: [1, 1, 1, 1, 1],
+    perSlotMinCount: [1, 1, 1, 1, 1, 1],
     slotNames: [
       'CS 2800 or CS 2802',
       'CS 3110',
       'CS 3410 or CS 3420',
+      'CS 3700 or CS 3780',
       'CS 4820',
       'CS 4410 or CS 4414',
     ],
   },
   {
     name: 'CS Electives',
     description:
-      'Three 4000+ CS electives each at 3 credits. CS 4090, CS 4998, and CS 4999 are NOT allowed.',
+      'Two 4000+ CS electives each at 3 credits. CS 4090, CS 4998, and CS 4999 are NOT allowed. CS 3700 or CS 3780 allowed if not used in CS core.',
```

Comment:
Great work with referencing the major information! This is definitely a much-needed PR on account of how many CS majors use CoursePlan.

One comment here though — can you add a `checkerWarning` that we do not check for whether or not you are allowed to take CS 3700 / CS 3780 towards this requirement?

In the old version, we filter out 4090, 4998, and 4999 because these are *all* not allowed, but in this case it is possible that it *could* be allowed and we wouldn't be checking for that.